react/components/molecules/tabs-text/tabs-text.js

```diff
@@ -75,21 +80,31 @@ export class TabsText extends PureComponent {
     };
 
     _style() {
-        return [styles.tabsText, this.props.style];
+        return [
+            styles.tabsText,
+            styles[`tabsText${capitalize(this.props.variant)}`],
+            this.props.style
+        ];
     }
 
+    _buttonStyle = () => {
+        return this.props.variant === undefined && styles.button;

```

Comment:
Do the same `capitalize` trick here. You're probably thinking something like "only apply style when it has no variant", but it must be "apply the general style and override with the variant".